src/index.html

```diff
@@ -11,9 +14,63 @@
       <div><!-- future highscore button? --></div>
     </core-navbar>
 
-    <a id="capidle-btn" href="./capidle/index.html">Click here to visit Capidle </a>
-    <a id="liedle-btn" href="./liedle/index.html">Click here to visit Liedle </a>
-    <a id="2048expert-btn" href="./2048-expert/index.html">Click here to visit 2048 Expert </a>
+    <div id="center-info">
+      <img id="title-img" src="./images/placeholder.png" alt="example" class="title-img">
+      <div id="capidle-details" class="game-details">
+        <p>Capidle</p>
+      </div>
+      <div id="liedle-details" class="game-details">
+        <p>Liedle</p>
+      </div>
+      <div id="a2048expert-details" class="game-details">    
+        <p>2048 Expert</p>
+      </div>
+    </div>
+  
+
+    
+
+    <a id="capidle-btn" href="./capidle/index.html" class="games-button" onmouseenter="capidleDetails()" onmouseleave="titleShow()">
+      <img src="./images/placeholder.png" alt="example" class="game-button-img">
+      <p id="capidle-btn-text">Capidle</p>
+  
+    </a>
+    <a id="liedle-btn" href="./liedle/index.html" class="games-button" onmouseenter="liedleDetails()" onmouseleave="titleShow()">
+      <img src="./images/placeholder.png" alt="example" class="game-button-img">
+      <p id="liedle-btn-text">Liedle</p>
+  
+    </a>
+    <a id="a2048expert-btn" href="./2048-expert/index.html" class="games-button" onmouseenter="a2048ExpertDetails()" onmouseleave="titleShow()">
+      <img src="./images/placeholder.png" alt="example" class="game-button-img">
+      <p id="a2048expert-btn-text">2048 Expert</p>
+  
+    </a>
     <script type="module" src="./main.js"></script>
+    <!-- TO IMPLEMENT <script type="module" src="./script.js"></script>-->
```

Comment:
Are you able to move the script contents into `script.js`? 
BTW the SonarLint extension can help detect these kinds of problems. 
![image](https://user-images.githubusercontent.com/78716153/184456608-889c1351-3848-4d95-8a00-0a047e4e0bf8.png)

Reply:
Will do, this was more for my next issue so I thought I'd push what I had first to close these two. Can move them if needed though.